pom.xml

```diff
@@ -9,10 +9,10 @@
 
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>redhat-dependency-analytics</artifactId>
-    <version>0.7.1-SNAPSHOT</version>
+    <version>${revision}-${changelist}</version>
```

Comment:
```suggestion
    <version>${revision}.${changelist}</version>
```

Reply:
Hi @NotMyFault 

I want to use the same schema as https://github.com/jenkinsci/aws-java-sdk-plugin/blob/master/pom.xml#L13

Or is it a strict rule to use a period instead of dash?
As a result, the version should be:  ${revision}-{changelist}
For example, 0.7.0-111.abcd

Reply:
> Or is it a strict rule to use a period instead of dash?

Valid examples can be found in the documentation only: https://www.jenkins.io/doc/developer/publishing/releasing-cd/#update-maven-pom-and-config under the section "Manually controlled prefix (optional)".

> For example, 0.7.0-111.abcd

You'll likely want to start with 1.x to prevent maven messing with lower increments than versions which already exist, as it has happened in the past.